buildSrc/src/main/java/Dependencies.kt

```diff
@@ -7,22 +7,22 @@ object Versions {
     const val coroutines = "1.3.3"
     const val android_gradle_plugin = "3.5.0"
     const val sentry = "1.7.10"
-    const val leakcanary = "2.3"
+    const val leakcanary = "2.4"
     const val leanplum = "5.4.0"
     const val osslicenses_plugin = "0.9.5"
     const val osslicenses_library = "17.0.0"
     const val detekt = "1.6.0"
 
     const val androidx_appcompat = "1.2.0-rc01"
-    const val androidx_biometric = "1.0.1"
+    const val androidx_biometric = "1.1.0-alpha01"
```

Comment:
Why the alpha version?

Reply:
This biometric library has just been kinda a pain and it seems like they've added some bug fixes/improvements (including those incorrect deprecation warnings)
https://developer.android.com/jetpack/androidx/releases/biometric#1.1.0-alpha01